### PR TITLE
RegExp: Case Insensitive

### PR DESCRIPTION
**Pull request by zerodouglas**
Currently when mentioning a user, there is no search for case insensitivity. This PR adds an extra modifier: 'i', which does that.

### CODE REVIEW

File: client/src/components/Message.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Message = ({ currentUser, error, errorMessage, children }) => {
   // Redux
   const { username } = useSelector(state => state.auth.user);
 
-  const regex = new RegExp("^@" + username + "$", "g");
+  const regex = new RegExp("^@" + username + "$", "i", "g");
```
Comment (kmr600) [inline client/src/components/Message.js:55]: Separating the "i" and "g" tags gets rid the "g" tag. To fix this, combine them together in a single quotation with no commas as follows:
`const regex = new RegExp("^@" + username + "$", "gi");`

Comment (kmr600) [inline client/src/components/Message.js]: Separating the "i" and "g" tags gets rid the "g" tag. To fix this, combine them together in a single quotation with no commas as follows:
`const regex = new RegExp("^@" + username + "$", "gi");`

**Review by kmr600 — CHANGES_REQUESTED** [review]

File: client/src/components/Message.js (hunk 1 above)
Comment (zerodouglas) [reply client/src/components/Message.js]: Regrets were made when committing at 5am with no sleep ha! Changes were made as requested in #2 commit.

**Review by kmr600 — CHANGES_REQUESTED** [review]
If you could remove the yarn.lock files from the commit, it will be ready to merge. For this project, we are using npm.

**Pull request closed without merging**